vm-setup/roles/v1aX_integration_test/templates/controlplane_centos.yaml

```diff
@@ -27,8 +27,9 @@ spec:
         extraArgs:
     preKubeadmCommands:
       - ifup eth1
-      - mv /tmp/akeys /home/centos/.ssh/authorized_keys
-      - chown centos:centos /home/centos/.ssh/authorized_keys
+      - useradd -m metal3
+      - mv /tmp/akeys /home/metal3/.ssh/authorized_keys
+      - chown metal3:metal3 /home/metal3/.ssh/authorized_keys
```

Comment:
you should not use commands for that. you should use the `User` feature of Kubeadmconfig : https://godoc.org/sigs.k8s.io/cluster-api/bootstrap/kubeadm/api/v1alpha3#User

Reply:
Done